app/Actions/Diagnostics/Checks/IniSettingsCheck.php

```diff
@@ -74,22 +74,22 @@ public function check(array &$errors): void
 				= 'Warning: exec function has been disabled. You may experience some error 500, please report them to us.';
 		}
 
-		if (empty(ini_get('user_agent'))) {
+		if (empty(\Safe\ini_get('user_agent'))) {
 			$errors[]
 				= 'Warning: user_agent for PHP is not set. You may experience problems when importing images via URL.';
 		}
 
-		if (!ini_get('assert.exception')) {
+		if (!(bool) \Safe\ini_get('assert.exception')) {
```

Comment:
```
 ------ --------------------------------------------------------------- 
  Line   Actions/Diagnostics/Checks/IniSettingsCheck.php                
 ------ --------------------------------------------------------------- 
  82     Only booleans are allowed in a negated boolean, string given.  
 ------ --------------------------------------------------------------- 
```

Reply:
`\Safe\ini_get()` returns a string. :sweat_smile:

Reply:
But then we should be really explicit and write what we mean, i.e. `\Safe\ini_get(...) === '1'` or whatever is needed. Simply, writing `(bool)` in front of it is ridiculous, because it does not make the type conversion any safer.